packages/truffle-compile/compilerSupplier/loadingStrategies/LoadingStrategy.js

```diff
@@ -4,8 +4,11 @@ const fs = require("fs");
 class LoadingStrategy {
   constructor(options) {
     const defaultConfig = {
-      versionsUrl: "https://relay.trufflesuite.com/solc/bin/list.json",
-      compilerUrlRoot: "https://relay.trufflesuite.com/solc/bin/",
+      compilerRoots: [
+        "https://solc-bin.ethereum.org/bin/",
+        "https://ethereum.github.io/solc-bin/bin/",
+        "https://relay.trufflesuite.com/solc/bin/"
+      ],
```

Comment:
Do we need this to be duplicated here?

Reply:
Removed the duplicate from `compilerSupplier`.